pkg/cm_accounts/cm_accounts.go

```diff
@@ -274,6 +275,7 @@ func (s *service) MintBookingToken(
 		expirationTimestamp,
 		price,
 		paymentToken,
+		false,
```

Comment:
Should we put a comment here so it's not forgotten what's this about? Hard coding things always makes me nervous. 😅

Reply:
Added 2 comments
 https://github.com/chain4travel/camino-messenger-bot/pull/63/files#diff-eaf90c995346ed89c73a0cea15889f6931bf8c34c5765bb16bd709038598fd38R271
https://github.com/chain4travel/camino-messenger-bot/pull/63/files#diff-eaf90c995346ed89c73a0cea15889f6931bf8c34c5765bb16bd709038598fd38R279